### PR TITLE
fix: tipping in medium

### LINKED ISSUE #1708: [BUG] Tipping on Medium not working
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Alby does not detect lightning addresses on medium any more e.g. https://maxirosson.medium.com/

### Screenshots [optional]

_No response_

### Steps To Reproduce

Visit: https://maxirosson.medium.com/

### Expected behavior

Alby detects the lightning address, turns blue and allows to send a payment to the person when you open the extension

### Alby information

the latest version

### Device information

Windows, Chrome

### Additional context

_No response_

### Are you working on this?

No

### PR DESCRIPTION
**Pull request by im-adithya**
### Describe the changes you have made in this PR

The site has changed since we updated our battery last time so updated it according to the new UI.

### Link this PR to an issue [optional]

Fixes #1708

### Type of change

(Remove other not matching type)

- `fix`: Bug fix (non-breaking change which fixes an issue)

### Screenshots of the changes [optional]

Screenshot 2022-11-07 at 2 56 29 PM

### How has this been tested?

Tested here https://maxirosson.medium.com/

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

**Review by escapedcat — APPROVED** [review]
ack

File: src/extension/content-script/batteries/Medium.ts [hunk 1]
```diff
@@ -4,65 +4,30 @@ import { findLightningAddressInText, setLightningData } from "./helpers";
...
-  if (!authorLinkSelector) return;
-
-  const shortBio = document.querySelector<HTMLParagraphElement>(
-    `${authorLinkSelector}~p:last-of-type`
-  )?.innerText;
-  if (!shortBio) return;
-
-  const match = findLightningAddressInText(shortBio);
+  const name =
+    document.querySelector<HTMLHeadingElement>(".pw-author-name")?.innerText;
```
Comment (bumi) [inline src/extension/content-script/batteries/Medium.ts:8]: can we be sure that there is no other element (like a comment) that has that class?

Comment (im-adithya) [reply src/extension/content-script/batteries/Medium.ts:8]: medium.com doesn't seem to use semantics at all and barely uses meaningful class names...
Also, this is only for the profile page so there shouldn't be any comments (and hence authors of any other kind) here as it's a showcase of the author's posts only.

Comment (bumi) [reply src/extension/content-script/batteries/Medium.ts:8]: ok, thanks.

**Pull request merged**